mri/operators/fourier/utils/processing.py

```diff
@@ -347,24 +273,16 @@ def estimate_density_compensation(kspace_loc, volume_shape, num_iterations=10):
         the kspace locations
     volume_shape: np.ndarray
         the volume shape
-    num_iterations: int default 10
-        the number of iterations for density estimation
+    implementation: str default 'pipe'
+        the implementation of the non-cartesian operator
+        can be 'pipe' which needs gpuNUFFT or 'cell_count'
+    kwargs: dict
+        extra keyword arguments to be passed to the density
+        compensation estimation
     """
-    from ..non_cartesian import NonCartesianFFT
-    from ..non_cartesian import gpunufft_available
-    if gpunufft_available is False:
-        raise ValueError("gpuNUFFT is not available, cannot "
-                         "estimate the density compensation")
-    grid_op = NonCartesianFFT(
-        samples=kspace_loc,
-        shape=volume_shape,
-        implementation='gpuNUFFT',
-        osf=1,
+    density_comp = get_density(implementation)(
+        kspace_loc,
+        volume_shape,
+        **kwargs
```

Comment:
This needs a backend argument as well (to select gpunufft). Otherwise you can also do 
`get_operator("gpunufft").pipe(kspace_loc, volume_shape)`

Reply:
Yes, that goes in kwargs. Cause I think at the moment for `cell_count` we dont have backend argument. So I didnt want to fixate on anything specifically